src/gei/Commands/GenerateScript/GenerateScriptCommandHandler.cs

```diff
@@ -199,11 +199,12 @@ exit 1
         return content.ToString();
     }
 
-    private string MigrateGithubRepoScript(string githubSourceOrg, string githubTargetOrg, string repo, string ghesApiUrl, string awsBucketName, string awsRegion, bool noSslVerify, bool wait, bool skipReleases, bool lockSourceRepo, bool keepArchive, string repoVisibility, string targetApiUrl)
+    private string MigrateGithubRepoScript(string githubSourceOrg, string githubTargetOrg, string repo, string ghesApiUrl, string awsBucketName, string awsRegion, bool noSslVerify, bool wait, bool skipReleases, bool lockSourceRepo, bool keepArchive, string repoVisibility, string targetApiUrl, bool useGithubStorage)
     {
         var ghesRepoOptions = ghesApiUrl.HasValue() ? GetGhesRepoOptions(ghesApiUrl, awsBucketName, awsRegion, noSslVerify, keepArchive) : null;
+        var useGithubStorageOption = useGithubStorage ? " --use-github-storage true" : string.Empty;
```

Comment:
using `true` at the end of a boolean option is redundant. This option is the same as other boolean flags.
```suggestion
        var useGithubStorageOption = useGithubStorage ? " --use-github-storage" : string.Empty;
```

Reply:
I would also pass this flag to the `GetGhesRepoOptions` method